packages/synapse-react-client/src/components/FullWidthAlert/FullWidthAlert.tsx

```diff
@@ -24,8 +25,11 @@ export type AlertButtonConfig = {
   | { href?: string }
 )
 
-export interface FullWidthAlertProps extends AlertProps {
-  variant: string
+export type FullWidthAlertVariant = 'warning' | 'info' | 'danger' | 'success'
```

Comment:
I decided to continue using `variant` as the prop for FullWidthAlert, since displayToast also uses `variant` as its prop to pass to FullWidthAlert. Since both FullWidthAlert and displayToast are used widely, updating the prop would require a larger set of changes. If we'd like to change the prop to match the MUI Alert prop (`severity`), then would tackle in a future focused PR